common/source/docs/common-rtf.rst

```diff
@@ -50,7 +50,6 @@ Planes from Partners
 VTOL/QuadPlanes from Partners
 =============================
 * `CUAV - Raefly <https://store.cuav.net/shop/raefly/>`__
-* `Event38 - E400 <https://event38.com/e400-vtol-drone/?v=0a10a0b3e53b>`__
```

Comment:
just update the link : https://event38.com/fixed-wing/e400-vtol-drone/

Reply:
done